.golangci.yml

```diff
@@ -1,10 +1,10 @@
 run:
-  skip-dirs:
-    - examples/*.go
```

Comment:
( I may be missing some background information here ). Are we not suppose to run the lint in example directory?

Reply:
Yes, atm. As multiple lint failures could be observed for the examples (considering their long history), we are willing to skip the lint checks atm. We can surely enable lint checks for examples too, once we revert the breaking changes. 
@AlexVulaj @jaitaiwan please feel free to correct me, if that's not how we want to proceed.